kivy/graphics/vertex_instructions_line.pxi

```diff
@@ -672,8 +670,15 @@ cdef class Line(VertexInstruction):
         '''
         def __get__(self):
             return self._points
+
         def __set__(self, points):
-            self._points = list(points)
+            tmp = []
+            for p in points:
```

Comment:
I haven't tested, but it looks like a slow implementation. I think mixing lists/tuples with plain values is not a priority. `points` could be either a plain list, a list of lists, or a list of tuples. This way, a faster algo could be used, something like what @kived proposed in the issue.